client/src/components/Indicators.tsx

```diff
@@ -279,9 +283,6 @@ class IndicatorsWithoutI18n extends Component<IndicatorsProps, IndicatorsState>
                               "form-radio" +
                               (this.state.activeTimeSpan === timespan ? " active" : "")
                             }
-                            onClick={() => {
-                              window.gtag("event", "month-timeline-tab");
-                            }}
```

Comment:
eeek! so we were tagging every time span change as "month" huh? glad we are fixing this now

Reply:
yeah, and also this is the one that was getting triggered twice for each click. Not sure why the duplication was happening, but moving the logging to `onChange` instead of `onClick` solved that.